llvm/test/Transforms/InstCombine/icmp-and-lowbit-mask.ll

```diff
@@ -0,0 +1,629 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+; RUN: opt < %s -passes=instcombine -S | FileCheck %s
+
+declare void @llvm.assume(i1)
```

Comment:
@nikic Do we still need to declare these intrinsic functions explicitly?

See also https://github.com/llvm/llvm-project/pull/78251.

Reply:
Don't seem to need it. Dropping in v2